src/CattyTests/Download/StoreProjectsDownloaderTests.swift

```diff
@@ -757,7 +757,7 @@ class StoreProjectsDownloaderTests: XCTestCase {
         let project = getStoreProjectMock()
```

Comment:
Please check whether we still need the "project mock" here. I think we don't. If we do not need it anymore, please remove the _getStoreProjectMock_ method, add a new member variable (e.g. _projectId_) and use this directly.